packages/design-system-docs/src/pages/startup/migrating-v2.md

```diff
@@ -64,6 +64,12 @@ In CMSDS v2, it will be changed to
 
 Imports to `@cmsgov/design-system-layout` or `@cmsgov/design-system-support` are now included in the main `scss` entry point. If you would like to only import specific styles, see the `@cmsgov/design-system` [`README.md`](https://github.com/CMSgov/design-system/blob/master/packages/design-system/README.md#file-structure) for more info on our SASS architecture.
 
+See the [theming page]({{root}}/startup/theming/) for more specific information on overriding the default Sass variables or CSS declarations.
+
+### Focus styles
+
+We released focus styles in [version 3.6.0](https://github.com/CMSgov/design-system/releases/tag/3.6.0) and with this V2 update we are disabling these gold focus styles by default and placing them in `beta` development as we continue to iterate on them. If you want to continue to use the gold focus styles please see the guidance on the [theming page]({{root}}/startup/theming#focus-settings)on how to enable them.
```

Comment:
What do you think about this to give more context?

## Focus styles

CMSDS focus styles will now be considered a `beta` feature, and will be turned off by default. Focus styles can be reenabled by setting the `$ds-include-focus-styles` build variable to true, but these styles will likely change in the future as we iterate on our design. See the [theming page](https://design.cms.gov/startup/theming/#focus-settings) for more information on how to customize or enable focus styles.

### Why the change?
When we first released focus styles in [version 3.6.0](https://github.com/CMSgov/design-system/releases/tag/3.6.0), our goal was to provide a recommended focus style implementation that would incorporate accessibility best practices, much of which was informed by [UKgov's research](https://design-system.service.gov.uk/get-started/focus-states/) and [USWDS's focus state implementation](https://designsystem.digital.gov/). While we spent 4 months developing, iterating, and gathering feedback before the release, we made several mistakes in the process. Due to communication and process shortcomings, our team wasn't able to fully consider how our design would impact existing focus styles, or whether or not our design was too specific for certain products. Our team assumed that the SCSS variables we provided were sufficient to adapt the focus styles for different use cases and we neglected to add an easy way to opt out of the new feature. 

In this release, we worked to correct these mistakes by putting the new styles under a flag and turning them off by default. In the future, we plan on iterating on our default styles to be more generalized and more in line with existing focus styles on CMS products. We will also be adding  additional mixins, variables, and documentation to make it easier for teams to. customize focus styles.